src/app/home/layouts/wallet-form/wallet-form.component.html

```diff
@@ -0,0 +1,6 @@
+<div class="container">
+  <i class="far fa-smile-beam" style="font-size: 7rem"></i>
```

Comment:
Inline style, shame on you xd 
Create a class like a decent human programmer.

Reply:
Xd